modules/apps/client-extension/client-extension-web/src/main/resources/META-INF/resources/admin/edit_theme_css.jsp

```diff
@@ -27,4 +27,21 @@ ThemeCSSCET themeCSSCET = editClientExtensionEntryDisplayContext.getCET();
 	<div class="form-text">
 		<liferay-ui:message key="this-css-replaces-clay-css" />
 	</div>
-</aui:field-wrapper>
\ No newline at end of file
+</aui:field-wrapper>
+
+<c:if test='<%= FeatureFlagManagerUtil.isEnabled("LPD-10773") %>'>
+	<aui:field-wrapper cssClass="form-group">
+		<react:component
```

Comment:
Sometimes the react component can destroy it's container, so make sure that the field-wrapper is still being rendered correclty.